libraries/Servo/src/Servo.cpp

```diff
@@ -82,9 +82,9 @@ uint8_t Servo::attach(int pin, uint16_t minUs, uint16_t maxUs)
 void Servo::detach()
 {
   if (_attached) {
+    pinMode(_pin, INPUT);
```

Comment:
INPUT leaves the pin in a high impedance state and subject to noise unless an external pull-up/down resistor is added to the circuit.

INPUT_PULLUP, however, will have a weak pull-to-VCC resistor placed on the pad.  Two potential issues are:
1. Is "always 1" a valid, safe state for servo control inputs?
2. What is the "typical" input impedance of a servo, and how does it compare to the (weak, can't seem to find the exact value) internal pullup?